tests/analyses/test_mtf.py

```diff
@@ -86,3 +91,33 @@ def test_fft_through_focus_mtf_sets_mtftype_correctly(self, simple_system, mtfty
         )
 
         assert result.Settings["Type"] == _FFT_THROUGH_FOCUS_MTF_MTFTYPE_EXPECTED_RETURN[mtftype]
+
+
+class TestHuygensMTF:
+    def test_can_run_fft_through_focus_mtf(self, simple_system):
+        result = huygens_mtf(simple_system)
+
+        assert result.Data is not None
+
+    def test_to_json(self, simple_system):
+        result = huygens_mtf(simple_system)
+
+        assert result.from_json(result.to_json())
+
+    @pytest.mark.parametrize(
+        "pupilsampling,imagesampling,imagedelta,mtftype,maximumfrequency",
+        [("64x64", "64x64", 0.0, "Modulation", 150.0)],
```

Comment:
Please add multiple sets of parameters; if possible, every parameter should be changed at least once. In the past, this has helped us to e.g. detect the problem with changing the MTF type for the FFT through focus MTF analysis.

Reply:
I have added three additional unit tests.

Reply:
I see you added the specification for the reference data as well, but that commit seems somehow to be lost. At least I can see only a single set of parameters when I open this PR in VS Code for the web. Can you look into this?

Reply:
Sorry, somehow I forgot to commit that change. This is addressed in https://github.com/MREYE-LUMC/ZOSPy/pull/55/commits/d566fe3322ca9ca320e498f5c30960894103b553. All unit tests pass on my machine.